plugins/dualGovernance/hooks/useCanCreateProposal.tsx

```diff
@@ -56,7 +56,7 @@ export function useCanCreateProposal() {
     abi: DaoAbi,
     functionName: "hasPermission",
     // where, who, permissionId, data
-    args: [PUB_DUAL_GOVERNANCE_PLUGIN_ADDRESS, address, keccak256(toHex("PROPOSER_PERMISSION")), "0x"],
+    args: [PUB_DUAL_GOVERNANCE_PLUGIN_ADDRESS, address!, keccak256(toHex("PROPOSER_PERMISSION")), "0x"],
```

Comment:
I would always pass a valid address

```suggestion
    args: [PUB_DUAL_GOVERNANCE_PLUGIN_ADDRESS, address ?? ADDRESS_ZERO, keccak256(toHex("PROPOSER_PERMISSION")), "0x"],
```

Reply:
I this case address should never be undefined due to:
```
query: {
  enabled: !!daoAddress && !!address
}
```

Reply:
I see. Still relying on an external component doesn't sound as compelling to me as ensuring that our code is internally consistent. My 5c